crates/nu-command/tests/commands/move_/column.rs

```diff
@@ -61,17 +59,16 @@ fn moves_columns_before() {
                 | move column99 column3 --before column2
                 | rename _ chars_1 chars_2
                 | get chars_2 chars_1
+                | flatten
                 | str trim
                 | str collect
             "#
         ));
 
-        assert!(actual.out.contains("ANDRES::JONATHAN"));
+        assert!(actual.out.contains("ADE:JNTANRS:OAHN"));
```

Comment:
I wonder why the output changed here after your fix. I would have guessed a similar output but just a slightly different pipeline (because of the changes in 0.60)

Reply:
In nu 0.444 after `get chars_1 chars_2` will get:
```
   0      A
   1      N
   2      D
   3      R
   4      E
   5      S
   6      :
   7      :
   8      J
   9      O
  10      N
  11      A
  12      T
  13      H
  14      A
  15      N
```
In nu 0.59 after `| get chars_1 chars_2| flatten` will get:
```
   0      A
   1      D
   2      E
   3      :
   4      J
   5      N
   6      T
   7      A
   8      N
   9      R
  10      S
  11      :
  12      O
  13      A
  14      H
  15      N
```
That's why the result differs, but that's not important we are testing column moving

Reply:
If we want to keep the result same as before, maybe we should check why `get` works not as expected, IMO

Reply:
There are a few tests like this as i recall and I couldn't get any of them to work like 0.44.